### ISSUE TITLE
No Devices Shown

### ISSUE THREAD

**Issue by firstusing** [koproductions-code/ha-groheblue#2]
Hey Konstantin,
first Thank you for the Work!

I do not get any Devices by now, the Hub has been created succesfully but then... nothing.

![integration_hub](https://github.com/user-attachments/assets/6035bc6e-df1b-47d0-afac-6c87c947b888)

Best regards

**Comment by firstusing** [comment]
Here the Log:

Logger: homeassistant.util.loop
Quelle: util/loop.py:136
Erstmals aufgetreten: 20:40:38 (1 Vorkommnisse)
Zuletzt protokolliert: 20:40:38

Detected blocking call to load_verify_locations with args (<ssl.SSLContext object at 0x7f8336f740d0>,) inside the event loop by custom integration 'groheblue' at custom_components/groheblue/__init__.py, line 58: devices = await client.get_devices() (offender: /usr/local/lib/python3.12/site-packages/httpx/_config.py, line 149: context.load_verify_locations(cafile=cafile)), please report it to the author of the 'groheblue' custom integration For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#load_verify_locations Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 189, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 641, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1990, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 742, in async_setup_locked await self.async_setup(hass, integration=integration) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 594, in async_setup result = await component.async_setup_entry(hass, self) File "/config/custom_components/groheblue/__init__.py", line 58, in async_setup_entry devices = await client.get_devices()

**Comment by koproductions-code** [comment]
Hey, thanks for sending the log. The error message you get generally shouldn't be responsible for breaking the integration. Because I only wrote the integration in the past few days, I haven't really implemented error logging. 

EDIT: Please first check what I suggested in my second comment. If that doesn't work, try this: 

To make this easier, would you be able to run this simple python script with your credentials and tell me if it works? Just to make sure that the reason isn't the package for connecting with the grohe api.

First:
```
pip install asyncio groheblue
````

Then:

```python
from groheblue import GroheClient

client = GroheClient("<EMAIL>", "<PASSWORD>")


async def main():
    await client.login()

    devices = await client.get_devices()  # get all devices
    print(devices)

    for device in devices:
        print(device.appliance_id)

if __name__ == "__main__":
    import asyncio

    asyncio.run(main())
```

**Comment by koproductions-code** [comment]
Also, a major disadvantage of the groheblue python package is that it currently only supports one location and one room in the Grohe app. Could you check in your Grohe App that only one location (house) and only one room exists?

**Comment by koproductions-code** [comment]
This issue should be resolved in the latest release (0.2.0). Feel free to open another issue if the problem persists.
